quadkey/bq/sql/ST_GEOGFROMQUADINT_BOUNDARY.sql

```diff
@@ -0,0 +1,27 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_QUADKEY@@.GEOJSONBOUNDARY_FROMQUADINT`
```

Comment:
Same, this is private, so I would call it as the main function and add the `__` prefix.